.github/ISSUE_TEMPLATES/bug_report.md

```diff
@@ -1,19 +1,25 @@
+---
```

Comment:
I think the directory needs to be singular ISSUE_TEMPLATE

Reply:
Yep. Fixed this.